tests/python/test_debug.py

```diff
@@ -110,11 +122,13 @@ def test_out_of_bound_with_offset():
     def func():
         x[0, 0] = 1
 
-    func()
+    with pytest.raises(RuntimeError):
```

Comment:
Just FYI, you may also match the error message, so that you don't capture a wrong runtime error instead:
```suggestion
    with pytest.raises(RuntimeError, match='out of bound'):
        func()
```
or even regex:
```suggestion
    with pytest.raises(RuntimeError, match=r'Accessing \(\d+, \d+\) out of bound'):
        func()
```